zUtilities/QuickSave.cpp

```diff
@@ -279,12 +279,12 @@ namespace GOTHIC_ENGINE {
   }
 
   void QuickSave::MenuLoop() {
-    if ( !Options::QuickSaveMode || zCMenu::inGameMenu || isLoading || !zinput->KeyToggled( Options::KeyQuickLoad ) ) return;
+    if ( !Options::QuickSaveMode || zCMenu::inGameMenu || isLoading ) return;
 
     oCSavegameInfo* info = gameMan->savegameManager->GetSavegame( iLastSaveSlot );
     if ( !info || !info->DoesSavegameExist() ) return;
 
-    LoadFromMainMenu();
+    if ( Options::LoadLastQuickSaveAfterGameStart || zinput->KeyToggled( Options::KeyQuickLoad ) ) LoadFromMainMenu();
```

Comment:
@Myszax It's probably a cleaner solution than mine, but wouldn't it also trigger when user opens menu during game by pressing Esc? How is it prevented? `zCMenu::inGameMenu` checks if user is in main menu, or `MenuLoop` runs only in main menu?